src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-group/cephfs-subvolume-group.component.ts

```diff
@@ -116,6 +119,17 @@ export class CephfsSubvolumeGroupComponent implements OnInit, OnChanges {
         icon: Icons.edit,
         click: () => this.openModal(true)
       },
+      {
+        name: this.actionLabels.NFS_EXPORT,
```

Comment:
`NFS Export` is not meaningful enough for an action IMO. It should be either `Create NFS Export` or `Add NFS Export`. and it should have a different icon since the `plus` icon is already used for Creating a group. Maybe a [server](https://forkaweso.me/Fork-Awesome/icon/server/) icon to indicate this is a server?

Reply:
that makes sense will apply these changes here itself https://github.com/ceph/ceph/pull/58303
And I think for icon probably [this](https://fontawesome.com/icons/file-export?f=classic&s=solid) could be a better option then showing server icon

Reply:
> if that icon is available in our forkawesome library, then you can probably use that.

I tried with `fileExport = 'fas fa-file-export'` but doesn't seem to work! Any other suggestion for it or probably for now going with Add button will work?

Reply:
@avanthakkar `Create NFS Export` can you do this as part of this PR itself and add a icon.

Reply:
@avanthakkar the reason its not working is because that specific icon is not available on the fork awesome library. You can search for the available icon in here, https://forkaweso.me/Fork-Awesome/ and see if something you can find there.